### PR TITLE
[SYCL] Enable local_accessor / host_accessor hash related tests.

### PR DESCRIPTION
**Pull request by mmoadeli**
[Issue 8332](https://github.com/intel/llvm/issues/8332) is addressed. Enable local_accessor / host_accessor hash related tests.

### CODE REVIEW

**Comment by AlexeySachkov** [comment]
Duplicate of #599 ?

**Review by keryell — APPROVED** [review]
Thanks. I love PR easy to review. :-)

File: tests/accessor/local_accessor_semantics.cpp [hunk 1]
```diff
@@ -50,8 +50,7 @@ struct storage {
 DISABLED_FOR_TEST_CASE(hipSYCL)
 ("local_accessor common reference semantics (host)", "[local_accessor]")({
 // Issue link https://github.com/intel/llvm/issues/8332
```
Comment (bader) [inline tests/accessor/local_accessor_semantics.cpp:52]: Shouldn't we remove the comment with the link to DPC++ issue?

Comment (mmoadeli) [reply tests/accessor/local_accessor_semantics.cpp:52]: I was not sure if we should remove it when it's also implemented for ComputeCPP

Comment (bader) [reply tests/accessor/local_accessor_semantics.cpp:52]: This issue points to DPC++ problem, not ComputeCPP, so it's better to remove to avoid confusion.

**Comment by bader** [comment]
One more thing.
As Alexey mentioned above, this patch duplicates https://github.com/KhronosGroup/SYCL-CTS/pull/599, which has been merged. Please, resolve merge conflicts.

**Review by ProGTX — APPROVED** [review]

**Comment by tomdeakin** [comment]
Please resolve merge conflicts.

File: tests/accessor/local_accessor_semantics.cpp (hunk 1 above)
Comment (mmoadeli) [reply tests/accessor/local_accessor_semantics.cpp]: sorry for missing this, it turned out not to be needed anymore.

**Comment by mmoadeli** [comment]
> Please resolve merge conflicts.

sorry for missing this, it turned out not to be needed anymore.

**Comment by bader** [comment]
@mmoadeli, can we close this PR?

**Comment by mmoadeli** [comment]
> @mmoadeli, can we close this PR?

yes, I close it.

**Comment by mmoadeli** [comment]
not needed anymore.

**Pull request closed without merging**